static/lang/re-en.json

```
@@ -1193,252 +1193,252 @@
         },
         "PersistentDamage": {
             "Acid1d4": {
-                "criticalSuccess": "[[/r (2*1d4)[persistent,acid]]]",
-                "success": "[[/r 1d4[persistent,acid]]]"
+                "criticalSuccess": "@Damage[(2*1d4)[persistent,acid]]",
+                "success": "@Damage[1d4[persistent,acid]]"
             },
             "Acid1d6": {
-                "criticalSuccess": "[[/r (2*1d6)[persistent,acid]]]",
-                "success": "[[/r 1d6[persistent,acid]]]"
+                "criticalSuccess": "@Damage[(2*1d6)[persistent,acid]]",
+                "success": "@Damage[1d6[persistent,acid]]"
             },
             "Acid2d6": {
-                "criticalSuccess": "[[/r (2*2d6)[persistent,acid]]]",
-                "success": "[[/r 2d6[persistent,acid]]]"
+                "criticalSuccess": "@Damage[(2*2d6)[persistent,acid]]",
+                "success": "*@Damage[2d6[persistent,acid]]"
             },
             "Acid2d6+2": {
-                "criticalSuccess": "[[/r (2*2d6+4)[persistent,acid]]]",
-                "success": "[[/r (2d6+2)[persistent,acid]]]"
+                "criticalSuccess": "@Damage[(2*(2d6+4))[persistent,acid]]",
```

```suggestion
                "criticalSuccess": "@Damage[(2*2d6+4)[persistent,acid]]",
```
This +4 is already doubled.